flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecGroupWindowAggregate.java

```diff
@@ -105,6 +105,15 @@
 @ExecNodeMetadata(
         name = "stream-exec-group-window-aggregate",
         version = 1,
+        consumedOptions = {
+            "table.optimizer.simplify-operator-name-enabled",
+            "table.local-time-zone",
+            "table.exec.state.ttl",
+            "table.exec.mini-batch.enabled",
+            "table.exec.mini-batch.size",
+            "table.generated-code.max-length",
```

Comment:
this we can also skip? code gen should not affect the topology? it should be safe to change this without side effects?